content/zh/docs/v3.3/installing-on-linux/cluster-operation/add-edge-nodes.md

```diff
@@ -170,39 +201,7 @@ KubeSphere 利用 [KubeEdge](https://kubeedge.io/zh/) 将原生容器化应用
     systemctl restart edgecore.service
     ```
 
-9. 边缘节点加入集群后，部分 Pod 在调度至该边缘节点上后可能会一直处于 `Pending` 状态。由于部分守护进程集（例如，Calico）有强容忍度，您需要手动 Patch Pod 以防止它们调度至该边缘节点。
```

Comment:
Why the original step 9 is removed which is necessary to prevent the cloud side workload from being deployed to the edge?  @Bettygogo2021 ?
@wenchajun has adjusted this script to make it right a few months ago, why it's removed?

Reply:
Because it has been moved to the previous section (At the beginning of this doc)